neonvm-runner/cmd/main.go

```diff
@@ -1032,17 +1032,17 @@ func buildQEMUCmd(
 	if err != nil {
 		return nil, fmt.Errorf("Failed to set up default network: %w", err)
 	}
-	qemuCmd = append(qemuCmd, "-netdev", fmt.Sprintf("tap,id=default,ifname=%s,script=no,downscript=no,vhost=on", defaultNetworkTapName))
-	qemuCmd = append(qemuCmd, "-device", fmt.Sprintf("virtio-net-pci,netdev=default,mac=%s", macDefault.String()))
+	qemuCmd = append(qemuCmd, "-netdev", fmt.Sprintf("tap,id=default,ifname=%s,queues=4,script=no,downscript=no,vhost=on", defaultNetworkTapName))
+	qemuCmd = append(qemuCmd, "-device", fmt.Sprintf("virtio-net-pci,mq=on,vectors=10,netdev=default,mac=%s", macDefault.String()))
```

Comment:
is it worth commenting that we should have `vectors = 2 * queues + 2` ?

and, worth linking to https://www.linux-kvm.org/page/Multiqueue ?